### ISSUE TITLE
Issue with spanish distribution of debugger

### ISSUE THREAD

**Issue by codehag** [devtools-html/debugger.html#4701]
https://support.mozilla.org/en-US/questions/1183851

A user has discovered that the debugger does not show files on his machine, and it seems to be a localization issue.

```
TypeError: str is null 
Traza de la pila:
KeyShortcuts.parseElectronKey@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/debugger/new/debugger.js:37806:8
on@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/debugger/new/debugger.js:37942:23
componentDidMount@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/debugger/new/debugger.js:24286:6
notifyAll@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react.js:842:9
close@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react.js:14599:5
closeAll@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react.js:19101:11
perform@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react.js:19048:11
batchedMountComponentIntoNode@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react.js:12357:3
perform@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react.js:19035:13
batchedUpdates@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react.js:10660:7
batchedUpdates@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react.js:16668:3
_renderNewRootComponent@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react.js:12551:5
_renderSubtreeIntoContainer@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react.js:12632:21
render@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react.js:12653:12
monkeyPatchRender/<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/shared/vendor/react-dom.js:128:14
renderRoot@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/debugger/new/debugger.js:3409:6
bootstrapApp@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/debugger/new/debugger.js:32502:5
onConnect resource://devtools/client/debugger/new/debugger.js:31465:7
step@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/debugger/new/debugger.js:31491:184
step/<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/debugger/new/debugger.js:31491:362

 Promise-backend.js:925
```

**Comment by bgrins** [comment]
Keyboard shortcut strings like these should not be localized: https://github.com/devtools-html/debugger.html/blob/34cbf8ce225c34745ccacbb348c40d91af56d77d/assets/panel/debugger.properties#L146. At the least we should have a localization note to not localize them

**Comment by jasonLaster** [comment]
hmm... i'm confused. i thought that shortcuts were localized?

**Comment by juliandescottes** [comment]
The CmdOrCtrl part should not be localized. We have warnings in other parts of the code:

https://searchfox.org/mozilla-central/rev/550148ab69b2879bfb82ffa698720ede1fa626f2/devtools/client/locales/en-US/sourceeditor.properties#113-115

> # Do not localize "CmdOrCtrl", "F", or change the format of the string.

A bit misleading I guess, since the "key" part can be localized.

**Comment by juliandescottes** [comment]
As discussed, by "CmdOrCtrl part should not be localized", I didn't mean it should be removed from the properties file, but that localizers should not attempt to translate "CmdOrCtrl".

On our side the most we can do is to update the localization comment. But this is also documented at https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Developer_tools_key_shortcuts
